Cargo.toml

```diff
@@ -43,23 +43,23 @@ bytes = "1.7"
 chrono = "0.4.38"
 clap = "4.5"
 cql2 = "0.3.0"
-duckdb = "=1.0.0"
+duckdb = "1.1.1"
 fluent-uri = "0.3.1"
 futures = "0.3.31"
 geo = "0.29.3"
 geo-types = "0.7.13"
-geoarrow = "0.3.0"
+geoarrow = "0.4.0-beta.3"
 geojson = "0.24.1"
 http = "1.1"
 jsonschema = { version = "0.22.3", default-features = false }
-libduckdb-sys = "=1.0.0"
+libduckdb-sys = "1.1.1"
 log = "0.4.22"
 mime = "0.3.17"
 mockito = "1.5"
 object_store = "0.11.0"
 openssl = { version = "0.10.68", features = ["vendored"] }
 openssl-src = "=300.4.1" # joinked from https://github.com/iopsystems/rpc-perf/commit/705b290d2105af6f33150da04b217422c6d68701#diff-2e9d962a08321605940b5a657135052fbcef87b5e360662bb527c96d9a615542R41 to cross-compile Python
```

Comment:
By the way if you're using rustls you don't need openssl. Do you have other dependencies that require it? Using rustls is much easier than openssl for building python wheels.

Reply:
Thanks! Opened an issue here: https://github.com/stac-utils/stac-rs/issues/579